uwblueprint/community-fridge-kw
# infra: setup staging environment

## Brief description. What is this change? 

### [Create Developer Staging Environment](https://www.notion.so/uwblueprintexecs/Create-Developer-Staging-Environment-9f88d77d2b8a47cdb89014dbe13db416)



## Implementation description. How did you make this change?
* Created a new staging site on firebase `communityfridgekw-staging.web.app`
* Created a new staging app on Heroku for backend server and db
* Created a new `staging` branch on github
* Updated github actions and secrets accordingly
* With these new changes, main is the production branch and once this PR is merged to main, new PRs will be merged to the staging branch after.



## Steps to test
1. Verify main production site https://communityfridgekw.web.app/ works fine
2. Verify staging site https://communityfridgekw-staging.web.app/ works fine. This wont work for now, but once something is merged on to staging branch, it will work. For now just test the preview URL.
3. Verify the backend and db for production and staging is different
4. Verify Preview URLs are working fine and is working on the staging backend




## Checklist
